Mathlib/Data/Complex/Exponential.lean

```diff
@@ -1175,6 +1175,71 @@ theorem abs_exp_sub_one_sub_id_le {x : ℂ} (hx : abs x ≤ 1) : abs (exp x - 1
     _ ≤ abs x ^ 2 * 1 := by gcongr; norm_num [Nat.factorial]
     _ = abs x ^ 2 := by rw [mul_one]
 
+lemma abs_exp_sub_range_le_exp_sub_range_abs (x : ℂ) (n : ℕ) :
+  abs (exp x - ∑ m ∈ range n, x ^ m / m.factorial)
+    ≤ Real.exp (abs x) - ∑ m ∈ range n, (abs x) ^ m / m.factorial := by
+  rw [← CauSeq.lim_const (abv := Complex.abs) (∑ m ∈ range n, _), Complex.exp, sub_eq_add_neg,
+    ← CauSeq.lim_neg, CauSeq.lim_add, ← lim_abs]
+  refine CauSeq.lim_le (CauSeq.le_of_exists ⟨n, fun j hj => ?_⟩)
+  simp_rw [← sub_eq_add_neg]
+  calc abs ((∑ m ∈ range j, x ^ m / m.factorial) - ∑ m ∈ range n, x ^ m / m.factorial)
+  _ ≤ (∑ m ∈ range j, abs x ^ m / m.factorial) - ∑ m ∈ range n, abs x ^ m / m.factorial := by
+    rw [sum_range_sub_sum_range hj, sum_range_sub_sum_range hj]
+    refine (IsAbsoluteValue.abv_sum Complex.abs ..).trans_eq ?_
+    congr with i
+    simp
+  _ ≤ Real.exp (abs x) - ∑ m ∈ range n, (abs x) ^ m / m.factorial := by
+    gcongr
+    exact Real.sum_le_exp_of_nonneg (by exact AbsoluteValue.nonneg abs x) _
+
+lemma abs_exp_le_exp_abs (x : ℂ) : abs (exp x) ≤ Real.exp (abs x) := by
+  convert abs_exp_sub_range_le_exp_sub_range_abs x 0 using 1 <;> simp
+
+lemma exp_bound_exp (x : ℂ) (n : ℕ) :
+    abs (exp x - ∑ m ∈ range n, x ^ m / m.factorial) ≤ abs x ^ n * Real.exp (abs x) := by
```

Comment:
The lemma name seems a bit generic here; how about something like `abs_exp_sub_sum_le_abs_mul_exp`?

Reply:
Oh right that name it totally bad, sorry, I forgot about it. The name you propose sounds good.
On the topic of bad names, a few lines above in the same file you'll find `exp_bound` and `exp_bound'`.